### PR TITLE
IA-1838: Show icon in completeness table when ou has multiple submissions

### PR DESCRIPTION
**Pull request by niconoe**
Since a previous PR, a new boolean flag (has_multiple_direct_submissions) was now returned by the API for each row in the completeness stats API

This PR shows a specific icon in the completeness table in cases this flag is set to true

Related JIRA tickets : [IA-1838](https://bluesquare.atlassian.net/browse/IA-1838)

## Self proof reading checklist

- [ ] Did I use eslint and black formatters
- [x] Is my code clear enough and well documented
- [ ] Are my typescript files well typed
- [ ] Did I add translations
- [ ] My migrations file are included
- [ ] Are there enough tests

## How to test

Assign multiple submissions to a given orgunit. Make it appear in the completeness table and make sure there is a new icon (+ help text) in the "direct counts" column

## Notes

PR still in draft because I can't seem to make eslint work on my machine!

[IA-1838]: https://bluesquare.atlassian.net/browse/IA-1838?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Comment by niconoe** [comment]
Update: still struggling with my ESLint configuration. That's the only thing missing, and I think this feature is useful (and was already communicated with the geo team).

So if someone could review it, happy eslint to the changed files and merge it that would be great!

File: hat/assets/js/apps/Iaso/domains/completenessStats/hooks/useCompletenessStatsColumns.tsx [hunk 1]
```diff
@@ -80,6 +82,8 @@ export const useCompletenessStatsColumns = (params: any) => {
                     <span>
                         {settings.row.original.forms_filled_direct ?? '--'}/
                         {settings.row.original.forms_to_fill_direct ?? '--'}
+
+                        {settings.row.original.has_multiple_direct_submissions ? <Tooltip title={formatMessage(MESSAGES.orgUnitHasMultipleSubmissions)}><AddCircleOutlineIcon fontSize="small" /></Tooltip> : ''}
```
Comment (quang-le) [inline hat/assets/js/apps/Iaso/domains/completenessStats/hooks/useCompletenessStatsColumns.tsx]: es-lint 😭

**Review by quang-le — CHANGES_REQUESTED** [review]
Color should be inline with other similar elements
The icon is not correctly aligned

Screenshot 2023-02-07 at 13 54 24

**Comment by quang-le** [comment]
Looks great in Chrome but misaligned in Firefox

Screenshot 2023-02-10 at 16 35 15

File: hat/assets/js/apps/Iaso/domains/completenessStats/hooks/useCompletenessStatsColumns.tsx [hunk 2]
```diff
@@ -77,10 +81,49 @@ export const useCompletenessStatsColumns = (params: any) => {
                 accessor: 'forms_filled_direct',
                 sortable: false,
                 Cell: settings => (
-                    <span>
-                        {settings.row.original.forms_filled_direct ?? '--'}/
-                        {settings.row.original.forms_to_fill_direct ?? '--'}
-                    </span>
+                    <Box>
+                        <div
```
Comment (beygorghor) [inline hat/assets/js/apps/Iaso/domains/completenessStats/hooks/useCompletenessStatsColumns.tsx]: You can use Box instead, and use `display`, `alignItems` props from `Box`, not sure de `verticalAlign` is usefull here...

File: hat/assets/js/apps/Iaso/domains/completenessStats/hooks/useCompletenessStatsColumns.tsx [hunk 3]
```diff
@@ -77,10 +81,49 @@ export const useCompletenessStatsColumns = (params: any) => {
...
+                            }`}
+                        </div>
+                        {settings.row.original
+                            .has_multiple_direct_submissions && (
+                            <Tooltip
+                                title={formatMessage(
+                                    MESSAGES.orgUnitHasMultipleSubmissions,
+                                )}
+                            >
+                                <div
```
Comment (beygorghor) [inline hat/assets/js/apps/Iaso/domains/completenessStats/hooks/useCompletenessStatsColumns.tsx]: Same here

File: hat/assets/js/apps/Iaso/domains/completenessStats/hooks/useCompletenessStatsColumns.tsx [hunk 4]
```diff
@@ -77,10 +81,49 @@ export const useCompletenessStatsColumns = (params: any) => {
...
+                                    style={{
+                                        display: 'inline-flex',
+                                        alignItems: 'center',
+                                        verticalAlign: 'middle',
+                                    }}
+                                >
+                                    <AddCircleOutlineIcon
```
Comment (beygorghor) [inline hat/assets/js/apps/Iaso/domains/completenessStats/hooks/useCompletenessStatsColumns.tsx:115]: I don't understand the meaning of this icon, there is no action or `Tooltip` linked to it

**Review by beygorghor — CHANGES_REQUESTED** [review]
You can split the `forms_filled_direct ` cell into a separate component i think + remarks

File: hat/assets/js/apps/Iaso/domains/completenessStats/hooks/useCompletenessStatsColumns.tsx (hunk 4 above)
Comment (quang-le) [reply hat/assets/js/apps/Iaso/domains/completenessStats/hooks/useCompletenessStatsColumns.tsx:115]: There is a tooltip:

Screenshot 2023-02-15 at 10 43 19

File: hat/assets/js/apps/Iaso/domains/completenessStats/hooks/useCompletenessStatsColumns.tsx (hunk 2 above)
Comment (quang-le) [reply hat/assets/js/apps/Iaso/domains/completenessStats/hooks/useCompletenessStatsColumns.tsx]: I can use the Box, but if I use the props i.o `style`, the layout breaks

File: hat/assets/js/apps/Iaso/domains/completenessStats/hooks/useCompletenessStatsColumns.tsx (hunk 3 above)
Comment (quang-le) [reply hat/assets/js/apps/Iaso/domains/completenessStats/hooks/useCompletenessStatsColumns.tsx]: Same as above. The verticalAlign is required here though

**Pull request merged**